alacritty.yml

```diff
@@ -234,6 +234,13 @@
     #  background: '#c5c8c6'
     #  foreground: '#1d1f21'
 
+  # Hyperlink URL preview bar
+  #
+  # Default to be the inverse of normal foreground/background.
+  #hyperlink_preview_bar:
+  #  background: '#c5c8c6'
+  #  foreground: '#1d1f21'
+
```

Comment:
Realistically is there any reason why the search and hyprlink preview bar should have different colors? I think unification might make sense.

Reply:
Simply because they are not the same thing (input bar v.s. output bar) and search bar color is under `search` field.
If we want to unify them into another field, I'm not sure what could be the suitable name. Just `bar_color` is a bad name.

Reply:
The name is a different question, but I definitely think combination and deprecation of the old search bar colors makes sense.

Reply:
Merged into `information_bar`